packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -2413,3 +2415,80 @@ const waitForSeq = async (
     };
     deltaManager.on("op", handleOp);
 });
+
+export interface ISummarizerRequestOptions {
+    cache: boolean,
+    reconnect: boolean,
+    summarizingClient: boolean,
+}
+
+/**
+ * Forms a function that will request a Summarizer.
+ * @param loaderRouter - the loader acting as an IFluidRouter
+ * @param lastSequenceNumber - the last sequence number (e.g., from DeltaManager)
+ * @param cache - use cache to retrieve summarizer
+ * @param summarizingClient - is summarizer client
+ * @param reconnect - can reconnect on connection loss
+ */
+export const formRequestSummarizerFn = (
+    loaderRouter: IFluidRouter,
+    options: ISummarizerRequestOptions,
```

Comment:
It's same before your PR, but I wonder why we allow to provide these options, as opposed to hard-coding values here. I'd think summaries would work incorrectly if we use anything other than { cache: false, reconnect: false, summarizingClient: true }. I can see maybe in future relaxing it around reconnect, but the structure of current code simply does not support it.
So, should we remove this arg and hard code all these values?

Reply:
You had originally recommended exposing these options, or at least "reconnect," as part of giving the server more control. At this point, I'm no longer exposing formRequestSummarizerFn (which could actually be private to SummaryManager at this point), so it definitely makes sense to hard code the options in there.

Reply:
Sorry if I'm sending conflicting messages. Good thoughts sometimes come too late :)
I'd say - let's expose reconnect flag if we validated (and added tests) to ensure it works with true. If not, we either should not expose it, or expose but assert that it has to be true today and some comment that may in future we will do work to allow passing false.

Reply:
No worries. I only brought it up to make sure there wasn't something you wanted to keep. If we're not getting any benefit from ISummarizerRequestOptions, I'm happy to delete it. It would be easy to add later.